src/signals.jl

```diff
@@ -338,6 +163,13 @@ Return the number of multichannel samples that fit within `duration` given `x.sa
 """
 sample_count(x, duration::Period) = TimeSpans.index_from_time(x.sample_rate, duration) - 1
 
+"""
+    sample_type(x)
+
+Return `julia_type_from_onda_sample_type(x.sample_type)`.
```

Comment:
I find this docstring to useless. Tried to make it a bit more useful:
```suggestion
    sample_type(x) -> Type

Return a Julia `Type` from a Onda sample name or type.
```

Reply:
Ha. 

The problem is that your suggestion is a good docstring for `julia_type_from_onda_sample_type`, but isn't quite right for this lil guy. What about:

```suggestion
    sample_type(x)

Return `x.sample_type` as an `Onda.LPCM_SAMPLE_TYPE_UNION` subtype. If `x.sample_type` is an Onda-specified `sample_type` string (e.g. `"int16"), it will be converted to the corresponding Julia type. If `x.sample_type <: Onda.LPCM_SAMPLE_TYPE_UNION`, this function simply returns `x.sample_type` as-is.
```

Reply:
Wayyyyyy better